test/support/postgrex_wal/psql.ex

```diff
@@ -0,0 +1,47 @@
+defmodule PostgrexWal.PSQL do
+  @moduledoc """
+  psql will use environment variables, suck as: PGDATABASE, PGHOST, PGPASSWORD, PGPORT and/or PGUSER
+
+  Some env variables (as below) has already set in Github traivs CI environment:
+  	PGUSER: postgres
+  	PGPASSWORD: postgres
+  	PG_SOCKET_DIR: /var/run/postgresq
+  """
+
+  def cmd(query) do
+    for q <- List.wrap(query) do
+      args = ["-c", q]
+      {output, status} = System.cmd("psql", args, stderr_to_stdout: true, env: pg_env())
```

Comment:
这样写你就不用环境变量就可以调整下了，不用和 psql 需要的环境变量对齐，没有下划线风格看着不是特别舒服


```suggestion
    env = pg_env()
    database_url = "postgres://#{env[:username}}:#{env[:password]}@#{env[:host]}:#{env[:port]}/#{env[:database]}"

    for q <- List.wrap(query) do
      args = [database_url, "-c", q]
      {output, status} = System.cmd("psql", args, stderr_to_stdout: true)
```

```elixir
  def pg_env do
    [
      username: System.get_env("PG_USERNAME", "postgres"),
      # snip
    ]
   end
```